go/my-app/main.go

```diff
@@ -26,12 +29,16 @@ func main() {
 		collectors.NewProcessCollector(collectors.ProcessCollectorOpts{}),
 	)
 
-	customStableMetric := promauto.With(reg).NewCounterVec(prometheus.CounterOpts{
-		Name: "my_app_custom_counter_total",
-		Help: "Custom counter metric for my app counts important things. It serves as an example " +
-				"of a very important metric that everyone is using.",
-	}, []string{"integer", "category", "fraction"})
-	customStableMetric.WithLabelValues("101", "AType", "1.22314").Inc()
+	switch *metricDefinition {
```

Comment:
Hope the intention is clear. This binary will help us showcase the case of e.g. same type of container runs with different versions where metric name changed between e.g.

`my-app -metric-source=generated@v0.1.0`
`my-app -metric-source=generated@v0.2.0`